docs/source/configuration/authorization.mdx

```diff
@@ -408,6 +416,192 @@ The response retains the initial request's shape but returns `null` for unauthor
 
 If _every_ requested field requires authentication and a request is unauthenticated, the router generates an error indicating that the query is unauthorized.
 
+<MinVersion version="1.34.0">
```

Comment:
Is this new? 😲 Is there a way we can use it to indicate that the YAML config up top is no longer required to enable these directives as of 1.34?

Reply:
We've had these since late July, but still need to use them comprehensively in the docs. (See https://github.com/apollographql/docs#minversion)

I think in the final version we should state that the directives are on by default. If you're using an earlier version, update if you can, otherwise you need to explicitly enable them.

Reply:
Updated `YAML config up top` section, that directives are enabled by default and here's an example configuration to disable them.